espflash/src/cli/mod.rs

```diff
@@ -133,6 +133,9 @@ pub struct FlashArgs {
     /// Logging format.
     #[arg(long, short = 'L', default_value = "serial", requires = "monitor")]
     pub log_format: LogFormat,
+    /// Minimum chip revision supported by image, in format: major * 100 + minor
+    #[arg(long, default_value = "0", value_parser = parse_chip_rev)]
```

Comment:
great we have the parse_chip_rev now 👍 we need to change the default to conform to the format